deployment/terraform/database.tf

```diff
@@ -66,7 +66,7 @@ resource "aws_db_parameter_group" "default" {
 }
 
 module "database" {
-  source = "github.com/azavea/terraform-aws-postgresql-rds?ref=2.6.0"
+  source = "github.com/azavea/terraform-aws-postgresql-rds?ref=2.5.0"
```

Comment:
This was downgraded to avoid enhanced monitoring IAM role name collisions across database instances.

Reply:
I kept this because I ran into an issue setting the interval again on production. I'm not worried about this feature being disabled on OAR.